CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.7.1
+
+Release date: August 30, 2017
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
```

Comment:
Would tweaking the text to `This release contains a high severity security update and it is highly recommended that users upgrade to this version.` work better?